src/database.rs

```diff
@@ -423,6 +515,14 @@ impl Database {
             NO_PARAMS,
         )?;
 
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS seshat_version (
```

Comment:
Maybe just `version`? The other table names aren't prefixed...

Reply:
Addressed by https://github.com/matrix-org/seshat/pull/15/commits/a8151284947f5346dc7ebc2a592bc6b6bab8d752.